styles.css

```diff
@@ -23,17 +24,30 @@
 
 .container {
   width: 100vw;
-  border: 100px;
   background: #191765;
   display: flex;
   align-items: center;
+  justify-content: center;
   text-align: center;
   border-bottom: 1px solid #646369;
   margin: 1px;
 }
 
+.grid {
+  display: grid;
+  grid-auto-rows: minmax(100px, auto);
+}
+
+/* .grid > * {
+  background: burlywood;
+}
+
+.grid > *:nth-child(odd) {
+  background: tomato;
+} */
```

Comment:
- [ ] Please can you remove any commented out code from your project, It helps to keep your code clean. And besides you can always use git versioning to see any previous code you wrote.

Reply:
Well resolved